File: javapatterns/src/main/java/org/iut/nantes/facade/avion/Avion.java
package org.iut.nantes.facade.avion;

public class Avion {
    private Facade facade;

    public Avion() {
        this.facade = new Facade();
    }

    public void decollage() {
        facade.decollage();
    }

    public void atterrissage() {
        facade.atterrissage();
    }
}

File: kotlinpatterns/src/main/kotlin/org/iut/nantes/facade/avion/Avion.kt
package org.iut.nantes.facade.avion

class Avion {
    private val facade: Facade
    fun decollage() {
        facade.decollage()
    }

    fun atterrissage() {
        facade.atterrissage()
    }

    init {
        facade = Facade()
    }
}